specification/keyvault/resource-manager/readme.md

```diff
@@ -151,6 +152,10 @@ input-file:
   - $(this-folder)/Microsoft.KeyVault/stable/2016-10-01/keyvault.json
   - $(this-folder)/Microsoft.KeyVault/stable/2016-10-01/providers.json
   - $(this-folder)/Microsoft.KeyVault/stable/2015-06-01/keyvault.json
+  - $(this-folder)/Microsoft.KeyVault/preview/2020-04-01-preview/keyvault.json
```

Comment:
These should be in their own tagged section. Just follow the format the rest use.

Reply:
Fixed